package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "patchcore",
-  "version": "1.13.2",
+  "version": "1.13.3",
```

Comment:
**Pull request etiquette tip:** It's generally a bad idea to increment the version of a module as part of a pull request since you have no idea if the PR will land before the next release. Also, depending on how the author manages their module, it might break their release process.

The latter is the case with patchcore as we use the `npm version major|minor|patch` command to handle versioning so that we get a clear commit message on each bump and a release tag automatically created.

Best to leave version bumps to the person publishing the module. ☮️